layouts/404.html

```diff
@@ -0,0 +1,11 @@
+{{ define "head-title" }}
```

Comment:
I'm not sure how to get the `hugo serve` functionality to show the 404 page. I was following [their directions](https://gohugo.io/templates/404/) for how it was supposed to work in GitHub pages.